### PR TITLE
Added loader size options and make the loader smaller

### PR DESCRIPTION
**Pull request by Thuhaa**
image

### CODE REVIEW

File: packages/svelte-undp-design/src/lib/Loader/Loader.stories.ts [hunk 1]
```diff
@@ -6,7 +6,19 @@ import Loader from './Loader.svelte';
...
+	tags: ['docsPage'],
+	argTypes: {
+		size: {
+			type: 'string',
+			description: 'Size of the loader',
+			defaultValue: 'medium'
+		},
+		parameters: {
+			size: {
+				values: ['small', 'large']
```
Comment (JinIgarashi) [inline packages/svelte-undp-design/src/lib/Loader/Loader.stories.ts:18]: Could you also add `medium` size?

File: sites/geohub/src/components/DataView.svelte [hunk 2]
```diff
@@ -330,19 +330,18 @@
     <div
       hidden={!$indicatorProgress}
       class="loader-container">
-      <Loader />
+      <Loader
+        class="m-auto"
+        size="small" />
```
Comment (JinIgarashi) [inline sites/geohub/src/components/DataView.svelte:335]: According to the screenshot you pasted, I feel new loader is a bit smaller for data tab. Personally, original size of loader is much better.

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
@Thuhaa Could you kindly paste other components' screenshot? I want to see how the loader looks like in other components.

- MapQueryInfoControl
- MiniMap
- TagFilter
- MapStyleCardList

File: packages/svelte-undp-design/src/lib/Loader/Loader.stories.ts (hunk 1 above)
Comment (Thuhaa) [reply packages/svelte-undp-design/src/lib/Loader/Loader.stories.ts:18]: 1. Minimap
image
2. MapStyleCardList 
image
3. TagFilter 
image
4. MapQueryInfo 
image

**Review by JinIgarashi — APPROVED** [review]
This looks good

**Pull request merged**